data/variants.txt

```diff
@@ -1473,108 +1668,62 @@ ship "Mule" "Mule (Hai Engines)"
 		"Cooling Ducts"
 		"Mass Expansion"
 		"Ramscoop"
+		
 		`"Bondir" Atomic Thruster`
 		`"Biroo" Atomic Steering`
 		`"Basrem" Atomic Steering`
 		"Hyperdrive"
 
-
-
 ship "Mule" "Mule (Hai Weapons)"
 	outfits
 		"Ion Cannon"
 		"Pulse Turret" 4
+		
 		"Geode Reactor"
 		"Hai Ravine Batteries"
 		"Hai Fissure Batteries"
 		"D67-TM Shield Generator"
 		"Water Coolant System"
 		"Mass Expansion" 3
 		"Ramscoop"
+		
 		"A370 Atomic Thruster"
 		"A375 Atomic Steering"
 		"Hyperdrive"
 
 
 
-ship "Leviathan" "Leviathan (Hai Engines)"
-	outfits
-		"Particle Cannon" 4
-		"Quad Blaster Turret" 2
-		"Chameleon Anti-Missile" 2
-		"Boulder Reactor"
-		"Hai Ravine Batteries"
-		"D67-TM Shield Generator"
-		"Liquid Helium Cooler"
-		`"Bondir" Atomic Thruster`
-		`"Bondir" Atomic Steering`
-		"Hyperdrive"
-
-
+# The Osprey (Alien Weapons) isn't used in any fleets, only in a special bounty mission.
```

Comment:
because I put that Osprey & the Behemoth (Heavy) in the lists while they were at like the bottom; these 2 don't appear in real fleets so was to make it clear
I can delete this & the Behemoth (Heavy) lines, those 2 + the human & alien indicator ones are the only actual comment lines I typed